lib/tasks/copy_database_to_s3.rake

```diff
@@ -3,45 +3,45 @@ require 'tempfile'
 
 namespace :scihist do
   desc """
-    Dump the live database and upload it to s3.
+    Dump the live database and upload it to s3:
 
-    BACKUP_AWS_ACCESS_KEY_ID=joe \
-    BACKUP_AWS_SECRET_ACCESS_KEY=schmo \
-    APP=scihist-digicoll \
-    rake scihist:copy_database_to_s3
+    bundle exec rake scihist:copy_database_to_s3
```

Comment:
maybe give an example of doing this with:

      eg from heroku where necessary config is defined:
      heroku run rake scihist:copy_database_to_s3

Reply:
Also could be worth nothing that this has to be run from where that has `pg_dump`, where it can be run without supplying username/password, to connect to database you want dumped -- it does not use the Rails database config, it just relies on `pg_dump`!

Really, this is meant for running on heroku, it might not really work right from other places, although could if things are set up right. I think probably the description should just say intended for running on heroku though, and mention that it's used by a nightly scheduled job?  Or even put in a `scihist:heroku:copy_database_to_s3` namespace, what do you think?